src/v3/src/transformer/selectAuthenticator/transformSelectAuthenticatorEnroll.ts

```diff
@@ -52,6 +54,7 @@ export const transformSelectAuthenticatorEnroll: IdxStepTransformer = (
   );
   uischema.elements.push({
     type: 'Description',
+    // TODO: re-visit, canSkip should not exist when use GenericRemediator
```

Comment:
`canSkip` or some indicator, should exist because it lets the user know they have optional authenticators they can enroll in but are not required to.

Reply:
I don't think it's even available in GenericRemediator